### PR TITLE
feat: better github guide

### LINKED ISSUE #2: Feat: Better GitHub guide
Hey I want tor request for better GitHub guide. The current one really lacks a lot of things.

### PR DESCRIPTION
**Pull request by BereketEngida**
This closes #2

### CODE REVIEW

**Comment by Bekacru** [comment]
Hey thanks for this pr can you please also add a section about github cli

**Comment by BereketEngida** [comment]
sure 👍

File: guides/github.md [hunk 1]
```diff
@@ -5,3 +5,35 @@ Github is code hosting platform for **version control** and collaboration.
...
+
+Clone is copying a remote repository to your local machine.
+
+## Branch
+
+Branches allows you to change a code push within a repository without affecting other branches including the main branch.
+
+## Pull Request
+
+Pull request is a request for a code change.
```
Comment (Bekacru) [inline guides/github.md:31]: Hey can we improve this line it's not really descriptive.

Comment (BereketEngida) [reply guides/github.md:31]: no prob

**Pull request merged**